website/docs/reference/events-logging.md

```diff
@@ -0,0 +1,106 @@
+...
+21:29:35.814348 [debug] [Thread-1  ]: On model.my_project.my_table: BEGIN
+```
+
+</File>
+
+## Structured logging
+
+When `json` [log formatting](global-configs#log-formatting) is enabled, dbt will output produce rich, structured log information which can be piped into monitoring tools for analysis, or to power applications with dbt metadata in real time.
+
+Each log line will have the following JSON properties:
+
+- `log_version`: Integer indicating version
+- `type`: Always `log_line`
+- `code`: A unique identifier for each event type
+- `ts`: When the log line was printed
+- `pid`: The process ID for the running dbt invocation which produced this log message
+- `msg`: The human-friendly log message. **Note**: This message is not intended for machine consumption. Log messages are subject to change in future versions of dbt, and those changes may or may not coincide with a change in `log_version`.
+- `level`: A string representation of the log level (`debug`, `info`, `warn`, `error`)
+- [`invocation_id`](invocation_id): A unique identifier for this invocation of dbt
+- `thread_name`: The thread in which the log message was produced, helpful for tracking queries when dbt is run with multiple threads
+- `data`: A dictionary containing programmatically accessible information about the log line. The contents of this dictionary vary based on the event type which generated this log message.
+- `node_info`: If applicable, a dictionary of human- and machine-friendly information about a currently running resource
+    - `node_name`
+    - `node_path`: File path to where this resource is defined
+    - `unique_id`: The unique identifier for this resource, which can be used to look up contextual information in a [manifest](artifacts/manifest-json)
+    - `resource_type`: model, test, seed, snapshot, etc.
+    - `materialized`: view, table, incremental, etc.
+    - `node_status`: Current status of the node, as defined in [the result contract](https://github.com/dbt-labs/dbt-core/blob/HEAD/core/dbt/contracts/results.py#L61-L74)
+    - `node_started_at`: Timestamp when node processing started
+    - `node_finished_at`: Timestamp when node processing completed
+    - `type`: Always `'node_status'`
+
+### Example
+
+```json
+{
+	"code": "Q033",
+	"data":
+	{
+		"description": "view model dbt_testing.name_list",
+		"index": 7,
+		"total": 7
+	},
+	"invocation_id": "30206572-f52f-4b91-af6d-d2b18fdbbbb8",
+	"level": "info",
+	"log_version": 1,
+	"msg": "7 of 7 START view model dbt_testing.name_list.............................. [RUN]",
+	"node_info":
+	{
+		"materialized": "view",
+		"node_finished_at": null,
```

Comment:
This shows up blank [in the page](https://deploy-preview-938--docs-getdbt-com.netlify.app/reference/events-logging#example), Do we want ti to say `null` or be blank?

Reply:
I see it saying `null` in the page preview!

Reply:
Oops was looking at the wrong thing!